Drive/rover_drive_system.hpp

```diff
@@ -237,6 +237,20 @@ class RoverDriveSystem : public sjsu::common::RoverSystem
   }
 
  private:
+
+  // https://github.com/SJSURoboticsTeam/urc-central-2021/issues/196
+  // S, D, T, L, R, B = Spin, Drive, Translate, Left, Right, Back.
```

Comment:
```suggestion
```
We should no longer need to have additional documentation on what the drive modes are as they clearly define themselves now. This is a big improvement to the code quality because needing to document what your code is doing is a sign of bad design as good code should read like good prose and be fully explicit.